src/Features/CSharp/Portable/Completion/CompletionProviders/AwaitCompletionProvider.cs

```diff
@@ -111,5 +48,28 @@ private static int GetSpanStart(SyntaxNode declaration)
                 _ => throw ExceptionUtilities.UnexpectedValue(declaration.Kind())
             };
         }
+
+        private protected override bool ShouldMakeContainerAsync(SyntaxToken token)
+        {
+            var declaration = GetAsyncSupportingDeclaration(token);
+            return declaration is not null && !declaration.GetModifiers().Any(SyntaxKind.AsyncKeyword);
+        }
+
+        private protected override CompletionItem GetCompletionItem(SyntaxToken token)
+        {
+            var shouldMakeContainerAsync = ShouldMakeContainerAsync(token);
+            var text = SyntaxFacts.GetText(SyntaxKind.AwaitKeyword);
+            return CommonCompletionItem.Create(
+                displayText: text,
+                displayTextSuffix: "",
+                rules: CompletionItemRules.Default,
+                Glyph.Keyword,
+                description: RecommendedKeyword.CreateDisplayParts(text, string.Empty),
```

Comment:
The description is one *existing* difference. I kept it the same as before (ie, VB has a tooltip and C# not):

![image](https://user-images.githubusercontent.com/31348972/126162061-89480afb-40eb-4d8d-829c-e3fd5aebc6af.png)

![image](https://user-images.githubusercontent.com/31348972/126162136-d6d3eeda-b66c-4e87-bc61-9f7ac4a5465b.png)

Let me know if you want me to move `VBFeaturesResources.Asynchronously_waits_for_the_task_to_finish` to the Core layer and have it in both languages.

Reply:
yeah, let's move to core and have same description.